packages/wrangler/src/init.ts

```diff
@@ -485,10 +486,12 @@ export async function initHandler(args: InitArgs) {
 					`/accounts/${accountId}/workers/services/${fromDashScriptName}/environments/${defaultEnvironment}/content`
 				);
 
-				await writeFile(
-					path.join(creationDirectory, "./src/index.ts"),
-					dashScript
-				);
+				for (const file of dashScript) {
+					await writeFile(
+						path.join(creationDirectory, `./src/${file.name}`),
```

Comment:
This will probably need a recursive mkdir on the dirname before doing a file write to work with nested files